public/docs/_examples/cb-i18n/ts/app/app.component.html

```diff
@@ -1,4 +1,34 @@
 <!--#docregion-->
 <!--#docregion i18n-attribute-meaning-->
 <h1 i18n="User welcome|An introduction header for this sample">Hello i18n!</h1>
-<!--#enddocregion i18n-attribute-meaning-->
\ No newline at end of file
+<!--#enddocregion i18n-attribute-meaning-->
+
+<!--#docregion i18n-ng-container-->
+<ng-container i18n>I don't output any element</ng-container>
+<!--#enddocregion i18n-ng-container-->
+
+<br />
+
+<!--#docregion i18n-with-comment-->
+<!--i18n: optional meaning|optional description -->
+I don't output any element either
```

Comment:
Ok so this is _another_ way to translate something w/o putting a DOM element in. Who cares?